src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/BigQueryBase.java

```diff
@@ -177,4 +177,27 @@ public void validateTheCmekKeyOfTargetBigQueryTableIfCmekIsEnabled(String cmek)
   public void waitForPageToRenderProperly() {
     SeleniumDriver.waitForPageToLoad();
   }
+
+  @Then("Enter BigQuery property {string} as macro argument {string}")
+  public void enterBigQueryPropertyAsMacroArgument(String pluginProperty, String macroArgument) {
+    enterPropertyAsMacroArgument(pluginProperty, macroArgument);
+  }
+
+  @Then("Enter BigQuery cmek property {string} as macro argument {string} if cmek is enabled")
+  public void enterBigQueryCmekPropertyAsMacroArgumentIfCmekIsEnabled(String pluginProperty, String macroArgument) {
+    String cmekBQ =  PluginPropertyUtils.pluginProp("cmekBQ");
+    if (cmekBQ != null) {
+      enterPropertyAsMacroArgument(pluginProperty, macroArgument);
+    }
+  }
+
+  @Then("Enter runtime argument value {string} for BigQuery cmek property key {string} if BQ cmek is enabled")
+  public void enterRuntimeArgumentValueForBigQueryCmekPropertyKeyIfBQCmekIsEnabled(String value,
+                                                                                   String runtimeArgumentKey) {
+    String cmekBQ = PluginPropertyUtils.pluginProp(value);
+    if (cmekBQ != null) {
```

Comment:
Also add a log when it is null indicating that `CMEK is disabled`

Reply:
Added log